index.bs

```diff
@@ -1104,6 +1112,29 @@ a minimum of friction, e.g., the user will not have to dig around in their pocke
 example of the latter, when the user is accessing the [=[RP]=] from a given client for the first time, they may be required to
 use a [=roaming authenticator=] which was originally registered with the [=[RP]=] using a different client.
 
+### User Verification Protection enumeration (enum <dfn enum>UserVerificationProtection</dfn>) ### {#user-verification-protection}
+
+<pre class="idl">
+    enum MatcherProtection {
+       "rich-operating-system-software", // need to break Rich-OS kernel in order to by-pass user verification for miusing the key
```

Comment:
You define "restricted operating environment" below, rather than "rich operating system software". You should use the same terms in both places. I like "restricted operating environment" better, although I'm not confident that's the best term for this.